python/adelphi/setup.py

```diff
@@ -1,5 +1,16 @@
 from setuptools import setup
 
+PY3 = sys.version_info[0] == 3
+
+dependencies = [
+    'cassandra-driver >= 3.24.0',
+    'click >= 7.1.2',
+    'PyGithub >= 1.45'
+    ]
```

Comment:
The versions specified here are the min versions currently used for Python 2.7.  The ">=" specifier allows Py3k to use larger versions as appropriate.

Reply:
Updated: shifted to compatible version specifiers as discussed in PEP-440